app/src/main/java/io/github/droidkaigi/confsched2017/di/OverlayViewComponent.java

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2017.di;
+
+import dagger.Subcomponent;
+import io.github.droidkaigi.confsched2017.di.scope.OverlayViewScope;
+import io.github.droidkaigi.confsched2017.view.customview.DebugOverlayView;
+
+/**
+ * @author KeithYokoma
+ */
+@OverlayViewScope
+@Subcomponent(modules = {OverlayViewModule.class})
+public interface OverlayViewComponent {
```

Comment:
@konifar Are you going to say how the "components" should be devided?
If so, I have the same question.